### DISCUSSION TITLE
TooltipComponent props (Javascript)

### DISCUSSION THREAD

**Discussion by haendzel** [gilbarbara/react-joyride#862] [Q&A] [answered]
Hello

I used from this example: https://codesandbox.io/s/react-joyride-with-form-4vezxs?file=/src/App.tsx:117-135
but i work in javascript only. How to write these props?

```
  const tooltipComponent = (TooltipRenderProps) => (
    <TooltipTutorial {...props} disabledButton={disableButton} />
  );
```

I have error: "ReferenceError: props is not defined".

**Comment by gilbarbara** [comment, marked as answer]
Hey @haendzel 

Use the `props` parameter.
In `props: TooltipRenderProps`, `props` is the parameter, `TooltipRenderProps` is the type.

**Reply by haendzel** [reply]
Sir, thank you so much.
